make/clean.mk

```diff
@@ -16,6 +16,9 @@ clean-users:
 	$(Q)-oc delete spaces --all --all-namespaces
 	$(Q)-oc wait --for=delete namespaces -l toolchain.dev.openshift.com/type
 
+clean-nstemplatetiers:
+	$(Q)-oc wait --for=delete nstemplatetier --all --all-namespaces
```

Comment:
Thanks for adding the wait, but you still need to call the `delete` right? it won't get deleted automatically

Reply:
ah, I misread the docs for the wait - I thought it would run the operation, too! 🤦🏼